.travis.yml

```diff
@@ -2,8 +2,8 @@ sudo: false
 language: go
 
 go:
- - 1.6.3
```

Comment:
How do we want to handle testing features that are only available in 1.7, like httptrace?